Jenkinsfile

```diff
@@ -1,10 +1,29 @@
 pipeline {
   agent any
   stages {
-    stage('error') {
+    stage('Unit') {
+      agent any
       steps {
-        sh 'tox -r'
+        sh 'tox'
```

Comment:
Tox should not recreate the virtual environment in every build?

Reply:
Since Jenkins create a new workspace for each build, it doesn't need to recreate the env because there's no env in the first place